src/bms_main.cpp

```diff
@@ -58,7 +58,9 @@ void bms_setup() {
   //
   // TODO(everyone): Experiment heavily on what is the smallest value accepted
   // by different boards without throwing the error.
-  relay.setCurrentRewriterCallback([](float amps) { return amps * 0.5; });
+  relay.setCurrentRewriterCallback([](float amps) {
+    return (Settings.coefficient * 2) * amps + Settings.offset;
```

Comment:
Just an FYI and not a call for action because this feature won't make it to the user visible release - the first time this code is loaded on the chip, the multiplier and offset will be populated with default values for floats which is 0 so essentially Owie will swallow all current reporting. This would go against your expectation of the default installation being a no-op.
It doesnt matter for now cuz it's literally just me and you playing with this stuff.